glean_parser/schemas/metrics.1-0-0.schema.yaml

```diff
@@ -28,6 +28,13 @@ definitions:
     pattern: "^[a-z_][a-z0-9_]{0,29}(\\.[a-z_][a-z0-9_]{0,29})*$"
     maxLength: 40
 
+  kebab_case:
+    type: string
+    # Bug 1601270; we allow 3 specific existing snake_cased ping names for now,
+    # but these special cases can be removed in 2020H2.
```

Comment:
Ah, why can these be removed in 2020H2?

Reply:
I'm assuming the number of clients sending pings with the snake case names will have died down to negligible levels by then, but I also plan to add monitoring in the data pipeline to tell us how often we're seeing snake_cased ping names, so that we can understand the impact if we remove support.

I'll add more to the comment here.